### PR TITLE
Feature/initial setup

### PR DESCRIPTION
**Pull request by jakeharding**
Sets up the project directory structure and provides an example on entry into Angular application from Django template.

### CODE REVIEW

File: repo_health/index/static/dist/HomeCtrl.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(function () {
```
Comment (bparish628) [inline repo_health/index/static/dist/HomeCtrl.js:1]: You don't need to self-invoke these.

File: repo_health/index/static/dist/app.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+(function () {
```
Comment (bparish628) [inline repo_health/index/static/dist/app.js:2]: same here.

File: repo_health/index/static/package.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "repo_health_frontend",
+  "version": "1.0.0",
+  "description": "Angular frontend for the repo health project.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (bparish628) [inline repo_health/index/static/package.json:8]: There are no dependencies for angular/ui router, so does the the index.html actually work?

**Review by bparish628 — APPROVED** [review]
I think we should do a component architecture for the UI. Take a look at https://toddmotto.com/exploring-the-angular-1-5-component-method/. I can update the angular code though for an example :smile:.

**Comment by bparish628** [comment]
I'm no python expert, but that side looks good from what I know 👍

File: repo_health/index/static/dist/app.js (hunk 2 above)
Comment (jakeharding) [reply repo_health/index/static/dist/app.js:2]: Do you not use IIFE to keep the global namespace clean?

File: repo_health/index/static/package.json (hunk 3 above)
Comment (jakeharding) [reply repo_health/index/static/package.json:8]: Yes, the index.html works.

**Comment by jakeharding** [comment]
Cool, I have another commit to add for migrations and touch up the readme before I merge it.  I'm cool with the component method so please feel free to change the Angular as you like.  I'm more worried about establishing a practice around the IIFE coding style.  I usually use this to keep the global namespace because the transpilers I have seen do the same.  If we are reviewing each others code, we need to know what to expect.

**Pull request merged**